### DISCUSSION TITLE
diff about headless and headful

### DISCUSSION THREAD

**Discussion by roughsoft** [getgauge/taiko#1757] [Development]
Today I find some diff about headless and headful when taiko script running.
In headful ,I click some text, for example "my menu",
then the spa app render menu,I click some menu for example "goto that" 
In headless, sometimes the script will reporting error ,could not find element "goto that".
sometimes the script will be ok.

**Comment by zabil** [comment]
There's not enough information to debug this problem so my guess is that the application renders different on headless mode.

Try using the [screenshot](https://docs.taiko.dev/api/screenshot/) api before clicking the "goto that" element and observe it when it fails. 

There are some checks done in the latest release 1.1.1 for element visibility you can try that. 
To help others help you, please share a project to replicate this problem.
